src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -50,6 +50,8 @@ public Path getCloudSdkPath() {
     } else {
       // home directory
       possiblePaths.add(System.getProperty("user.home") + "/google-cloud-sdk");
+      // usr directory
+      possiblePaths.add("/usr");
```

Comment:
Thanks @elharo 

Checked with my Ubuntu 14.04.5 installation, link points to:
/usr/bin/gcloud -> /usr/lib/google-cloud-sdk/bin/gcloud

I added just "/usr" because "/bin" is added after programmatically. Path must match SDK directory, and not the bin directory.

As you said, correct possible path should be /usr/lib/google-cloud-sdk/